src/main/java/javax/measure/unit/NonSI.java

```diff
@@ -196,6 +220,20 @@ public static NonSI getInstance() {
      */
     public static final Unit<Length> COMPUTER_POINT = PIXEL;
 
+
+    /**
+     * New unit of length equal to <code>>meter</code>
+     */
+    private static final Unit<Length> GIGA_MICRON = new AlternateUnit<>("gmicron",
+            METER);
```

Comment:
Why is this needed?

Reply:
You cannot directly give alternate names to non-base units (such as micrometer/micron). But you can make a non-sense alternate of the base unit, and derive the aliased name from that non-sense unit.

Reply:
Side note: we would probably want a different symbol than gmicron. These units can unfortunately be instantiated in Seeq, so we want to give them names that won't occur by accident.

Reply:
made AlternateUnit public and moved this implementations into ScalarUnits class